rickled/__init__.py

```diff
@@ -122,6 +132,13 @@ def from_rickle(self, rickle, cls):
                                             return_function=True, is_method=True, includes_self_reference=_includes_self_reference)
 
                     obj.__dict__.update({_name: partial(f, obj)})
+                if value['type'] == 'action':
+                    # print('loading action', value)
+                    from guidance_strategies.action import ConditionalGuidanceAction
```

Comment:
@FabianSperrle This looks like something specific from your use case, and your own project classes. If so, please remove.